airgapped-rest_api/retriever_reader.yml

```
@@ -0,0 +1,34 @@
+components:
+  - name: DocumentStore
+    type: ElasticsearchDocumentStore
+  - name: Retriever
+    params:
+      document_store: DocumentStore
+    type: TfidfRetriever
+  - name: reader
```

All other components start with a capital letter, let's be consistent:
```suggestion
  - name: Reader
```